doc/neps/roadmap.rst

```diff
@@ -18,25 +18,19 @@ may include (among other things) interoperability protocols, better duck typing
 support and ndarray subclass handling.
 
 The key goal is: *make it easy for code written for NumPy to also work with
-other NumPy-like projects.* This will enable GPU support via, e.g, CuPy or JAX,
+other NumPy-like projects.* This will enable GPU support via, e.g, CuPy, JAX or PyTorch,
 distributed array support via Dask, and writing special-purpose arrays (either
 from scratch, or as a ``numpy.ndarray`` subclass) that work well with SciPy,
-scikit-learn and other such packages.
+scikit-learn and other such packages. A large step forward in this area was
+made in NumPy 2.0, with adoption of and compliance with the array API standard
+(v2022.12, see :ref:`NEP47`). Future work in this direction will include
+support for newer versions of the array API standard, and adding features as
+needed based on real-world experience and needs.
 
-The ``__array_ufunc__`` and ``__array_function__`` protocols are stable, but
-do not cover the whole API.  New protocols for overriding other functionality
-in NumPy are needed. Work in this area aims to bring to completion one or more
-of the following proposals:
-
-- :ref:`NEP30`
-- :ref:`NEP31`
-- :ref:`NEP35`
-- :ref:`NEP37`
-
-In addition we aim to provide ways to make it easier for other libraries to
-implement a NumPy-compatible API. This may include defining consistent subsets
-of the API, as discussed in `this section of NEP 37
-<https://numpy.org/neps/nep-0037-array-module.html#requesting-restricted-subsets-of-numpy-s-api>`__.
+In addition, the ``__array_ufunc__`` and ``__array_function__`` protocols
```

Comment:
My thought at the time those went in was that we needed to do something quickly to slow the fragmentation of the ecosystem and demonstrate the utility of interoperability. Standardization was the long term solution, but I figured it would take 5 - 10 years, and here we are, six years later.